Frontend/ui-customer/src/RemoteConfigs/Utils.purs

```diff
@@ -25,4 +28,25 @@ safetyBannerVideoConfigData :: String -> String -> Array SafetyVideoConfig
 safetyBannerVideoConfigData city language = do
     let config = fetchRemoteConfigString ("safety_banner_videos_" <> language)
         value = decodeForeignObject (parseJSON config) $ defaultRemoteConfig []
-    getCityBasedConfig value city
\ No newline at end of file
+    getCityBasedConfig value city
+
+pickupInstructions :: String -> String -> String -> Array PickupInstructions
+pickupInstructions placeName gateName language = do
+    let config = fetchRemoteConfigString ("pickup_instructions_" <> language)
+        pickupPlaces = decodeForeignObject (parseJSON config) defPlaces
```

Comment:
handle it in safe way using `decodeJson` by defining function with Maybe return type

Reply:
Maybe is not needed, It's already returning empty array